apps/web/src/components/DepositAndBorrow/Deposit/CurrentDepositTable.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  BoxLayout,
+  Button,
+  CellProps,
+  ColumnLayout,
+  Icon,
+  Table,
+  Text,
+  Toggle,
+} from '@bloxifi/ui'
+import React, { FunctionComponent } from 'react'
+
+import { DepositTitleBox } from './DepositTitleBox'
+
+import { WalletContainer } from '@/containers/WalletContainer'
+
+export const CurrentDepositTable: FunctionComponent = () => {
+  const {
+    state: { userReserves },
+  } = WalletContainer.useContainer()
+
+  const columns = {
+    assets: {
+      header: 'Assets',
+      Cell: ({ data: { symbol, icon } }: any) => (
```

Comment:
@aleksaToljic  Here I got an error for Icon component because it doesn't accept name as "unknown" that we have in CellProps. (Type 'unknown' is not assignable to type 'string') Not sure how we can fix this

Reply:
CurrentBorrowTable and CurrentDepositTable will be completed after Borrow and Deposit modals